test/functional/api/api-3rd-party-checkupdate.bats

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bats
+
+# Author: Castulo Martinez
+# Email: castulo.martinez@intel.com
+
+load "../testlib"
+
+test_setup() {
+
+	create_test_environment "$TEST_NAME"
+
+	create_third_party_repo -a "$TEST_NAME" 10 staging repo1
+	create_bundle -L -t -n test-bundle1 -f /foo/file_1 -u repo1 "$TEST_NAME"
+	create_version -p "$TEST_NAME" 20 10 staging repo1
+	update_bundle "$TEST_NAME" test-bundle1 --update /foo/file_1 repo1
+
+	create_third_party_repo -a "$TEST_NAME" 10 staging repo2
+	create_bundle -L -t -n test-bundle2 -f /bar/file_2 -u repo2 "$TEST_NAME"
+
+}
+
+@test "API008: 3rd-party checkupdate" {
+
+	run sudo sh -c "$SWUPD 3rd-party check-update $SWUPD_OPTS --quiet"
+
+	assert_status_is "$SWUPD_OK"
+	expected_output=$(cat <<-EOM
+		[repo1]
+		20
+		[repo2]
```

Comment:
I am unsure about this. Should we print the header [repo2] if there's no update?

Reply:
It would look nicer if we didn't, but I think is a limitation we have. I think it would be very difficult to not print it, I couldn't think of a way of doing it without doing a massive refactoring.